components/Toolbox.vue

```diff
@@ -38,12 +39,13 @@
         ></i>
 
         <!-- PANEL TITLE -->
-        <span class="panel-title">{{ state.title }}</span>
+        <span class = "panel-title">{{ state.title }}</span>
 
         <!-- TOGGLE EDITING -->
         <i
           v-disabled              = "editDisabled"
           @click.stop             = "toggleEditing"
+          style                   = "max-height: 35px; margin-left: auto"
```

Comment:
@volterra79 `max-height` is necessary? (show me a use case)

Reply:
@Raruto you are right. it isn't necessary. My mistake